app/views/registration_wizard/shared/_question_page.html.erb

```diff
@@ -19,7 +19,8 @@
         render(
           "registration_wizard/shared/questions/#{form.question.type}",
           form: f,
-          question: form.question
+          question: form.question,
+          fieldset_legend_attributes: form.question.try(:fieldset_legend_attributes)
```

Comment:
`form.question` is already being passed in, so we don't need to pass this in separately and we'd be able to grab this only where we know it's needed, i.e. inside `_radio_button_group.html.erb`, and not need to use try, might be a little cleaner to only reference this where necessary